coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +260,284 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
             self.assertEqual(
                 conditionally_update_storage_allocation_status_mock.call_count, 2
             )
+
+
+class TestIngestQuotasWithDailyUsages(TestCase):
+    def setUp(self) -> None:
+        self.original_storage_path = os.environ.get("STORAGE2_PATH")
+        self.STORAGE2_PATH = "/storage2/fs1"
+        os.environ["STORAGE2_PATH"] = self.STORAGE2_PATH
+
+        self.client = Client()
+        build_data = build_models()
+
+        self.project = build_data["project"]
+        self.user = build_data["user"]
+
+        self.quotas = self.__get_qumulo_quota_data(
+            get_mock_quota_data(self.STORAGE2_PATH)
+        )
+
+        self.status_active = AllocationStatusChoiceFactory(name="Active")
+        self.status_ready_for_deletion = AllocationStatusChoiceFactory(
+            name="Ready for Deletion"
+        )
+
+        for index, (path, value) in enumerate(
+            get_mock_quota_data(self.STORAGE2_PATH).items()
+        ):
+            if "exclude" in path:
```

Comment:
@harterjwustl 
I would have kept the the collection for creating the Coldfront allocations separated from the QUMULO mock data. 

On the other hand, if we are keeping the  get_mock_quota_data as implemented, I would create a method get_coldfront_seed_data (or something like that) that would have filter out the data. This way the exclusion is contained in one method.

Reply:
I'll look into implementing the second option.  I have to disagree on the first.  The QUMULO mock data is easy enough to build, and keeping them separate reuses so much shared data and allows for easy errors.  changing a character in the path or quota limit of one but not the corresponding other will break tests pretty quickly.  Furthermore, I don't *really* think we should be using actual wustlkey data here, but wasn't prepared to change it at the moment.

Maybe I'll move the mock data builder into the mock data file.